### PR TITLE
Version19

### PR DESCRIPTION
**Pull request by paying4529-zz**
- schmitt filter

### CODE REVIEW

**Comment by p9n** [comment]
wait.. is this same as PR #21?

**Comment by paying4529-zz** [comment]
Sorry.... I forgot to push...

**Comment by paying4529-zz** [comment]
discard schmitt filter
change to skip debouncing

=> this PR is built on version20

File: hummingbird.py [hunk 1]
```diff
@@ -420,7 +420,7 @@ def measure_both_scl_sda(self):
           )
           self.scl_falling_edge += 1
           scl.low_start = scl.i_30p
-          scl_skip = i + 2e-8 / self.sampling_period
+          scl_skip = i + 5e-8 / self.sampling_period
```
Comment (p9n) [inline hummingbird.py]: Give this constant a name, for example: `T_SP = 5e-8`

**Review by p9n — APPROVED** [review]

**Pull request merged**